projects/transfer-journey/src/lib/transfer-journey.module.ts

```diff
@@ -0,0 +1,53 @@
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { ReactiveFormsModule } from '@angular/forms';
+import { provideRoutes, Route, RouterModule } from '@angular/router';
+import { AccountSelectorModule, ButtonModule, CurrencyInputModule, InputValidationMessageModule, ModalModule } from '@backbase/ui-ang';
+import { MakeTransferFormComponent } from './components/make-transfer-form.component';
+import { MakeTransferSummaryComponent } from './components/make-transfer-summary.component';
+import { TRANSLATIONS } from './constants/dynamic-translations';
+import { TransferJourneyComponent } from './transfer-journey.component';
+import { MakeTransferComponent } from './views/make-transfer.component';
+
+const defaultRoute: Route = {
+  path: '',
+  component: TransferJourneyComponent,
+  children: [
+    {
+      path: '',
+      redirectTo: 'make-transfer',
+      pathMatch: 'full'
+    },
+    {
+      path: 'make-transfer',
+      component: MakeTransferComponent,
+      data: {
+        title: TRANSLATIONS.makeTransferTitle,
+      }
+    }
+  ]
+}
+
+@NgModule({
+  declarations: [TransferJourneyComponent, MakeTransferComponent, MakeTransferFormComponent, MakeTransferSummaryComponent],
+  imports: [
+    CommonModule,
+    RouterModule,
+    ButtonModule,
+    ModalModule,
+    CurrencyInputModule,
+    AccountSelectorModule,
+    InputValidationMessageModule,
+    ReactiveFormsModule,
+  ],
+  exports: [TransferJourneyComponent]
+})
+export class TransferJourneyModule {
+  static forRoot(data: { [key: string]: any; route: Route } = { route: defaultRoute }) {
+    return {
+      ngModule: TransferJourneyModule,
+      providers: [provideRoutes([data.route])],
+    };
```

Comment:
Another suggestion is to pass journey build-time config through method parameters instead of using token outside the journey:

```suggestion
  static forRoot(config: Partial<MakeTransferJourneyConfiguration> = {}) {
    return {
      ngModule: TransferJourneyModule,
      providers: [{
        provide: MakeTransferJourneyConfigurationToken,
        useValue: config
      }],
    };
```

Reply:
so at the end that was a decision made by business and retail vs to do it like that